DataBase/dataBase.py

```diff
@@ -26,22 +26,19 @@ def add_user(username, password):
 def add_camera(address, nom):
 
     with get_db_connection() as (cursor, conn):
-        query = "select * from cameras"
-        cursor.execute(query)
-
-        result = cursor.fetchall()
-        number_cam = 0
-        for rows in result:
-            number_cam += 1
-
-        print("Number cameras = ", number_cam)
+        try:
```

Comment:
adding exception handling to database functions is mandatory to avoid any crash in case of an incorrect SQL query plus the privilege of providing meaningful logging information